src/app/attendance/components/AttendanceList.tsx

```diff
@@ -0,0 +1,154 @@
+"use client";
+
+import React, { useEffect, useMemo } from "react";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+
+import { useToast } from "@/hooks/use-toast";
+import { AttendanceStatus } from "@/types";
+import { UserStatusBadge } from "@/common/StatusBadge/UserStatusBadge";
+import { StandardPage } from "@/common/StandardPage/StandardPage";
+import { Plus } from "lucide-react";
+import MemberTableActionRender from "@/app/members/components/MemberActionRender";
+
+interface Attendance {
+  member: string;
+  id: number;
+  startTime: string;
+  endTime: string;
+  status: AttendanceStatus;
+}
+interface AttendanceProps {
+  attendance: {
+    data: Attendance[];
+  };
+}
```

Comment:
@Mubashar1009  What's going on? Why is it so difficult to complex a simple attendance interface?

Reply:
fixed